configure.ac

```diff
@@ -541,6 +541,7 @@ WX_ARG_WITH(libnotify,     [  --with-libnotify        use libnotify for notifica
 WX_ARG_WITH(opengl,        [  --with-opengl           use OpenGL (or Mesa)], wxUSE_OPENGL)
 WX_ARG_WITH(xtest,         [  --with-xtest            use XTest extension], wxUSE_XTEST)
 WX_ARG_WITH(nanosvg,       [  --with-nanosvg          use NanoSVG for rasterizing SVG], wxUSE_NANOSVG)
+WX_ARG_WITH(lunasvg,       [  --with-lunasvg          use LunaSVG for rasterizing SVG], wxUSE_LUNASVG)
```

Comment:
You've only modified this `configure.ac` file. You'll need to run `autogen.sh` to update the `configure` file.

Reply:
I haven't used `configure` in a while, but I think it generates its own `setup.h` from `setup.h.in`.

I'll try it with lunasvg and report back.

Did you manually update the setup files or use `build/update-setup-h`?
(see `docs/contributing/how-to-add-new-wxUSE_XXX.md`)